docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+knesset:
+  image: doodyparizada/open-knesset
```

Comment:
we should probably change this to the hasadna organization on dockerhub. @noamelf permissions?